docs/website/contents/explanation/consensus-protocol.md

```diff
@@ -176,4 +191,79 @@ Although headers are already validated by the `ChainSync` client, they are re-va
+| `opcert` number | 2 | j | 1 |
+| VRF             | 3 | 2 | 1 |
+
+Lower-case letters stand for arbitrary values.
+
+In this example we have:
+
+- `B` is preferred over `A`, since `B` has lower VRF than `A`.
+- `C` is preferred over `B`, since `C` has lower VRF than `B`.
+- However `C` is **not** preferred over `A`, since they have the same issuer and slot, and therefore we prefer the chain with the highest `opcert` number (2), therefore `A` is preferred over `C`.
+
+Also, the `RestrictedVRFTiebreaker` flavour breaks the transitivity of chain ordering. To see this consider the following example where chains `D`, `E`, and `F` have the same length and different issuers, and assume `maxDist = 5` slots:
+
+|              | D | E | F |
+| ------------ | - | - | - |
+| Slot         | 0 | 3 | 6 |
+| VRF          | 3 | 2 | 1 |
+
+
+We have that:
+- `E` is preferred over `D`, since `E` has lower VRF than `D` and `|0 - 3| <= 5`.
+- `F` is preferred over `E`, since `F` has lower VRF than `E` and `|3 - 6| <= 5`
+- However, `D` is **not** preferred over `F`, but instead `D` and `F` are equally preferred since `|0 - 6| > 5` which implies that the VRF values of `D` and `F` are not used.
+
+Despite the non-transitivity, the fundamental Consensus properties, such as [Common Prefix](TODO-ref!), are not affected. This is because the primary factor for chain selection for a [caught-up](TODO-ref!) node remains the chain length, with longer chains always being preferred.
+The Ouroboros Praos authors make no assumptions about how nodes resolve ties.
+However, a non-transitive chain ordering brings the following complications:
+
+- **Implementation**: The use of `sortBy` from [`base`](https://hackage.haskell.org/package/base) in chain selection, which is typically expected to work with transitive relations, could become a concern. While preliminary property tests suggest it works for the current non-transitive order, there's a potential risk that future GHC implementations might behave in unexpected ways.
+- **Reasoning**: The non-transitive order can be very confusing for reasoning about anything related to chain order, as transitivity is an implicitly assumed property when working with order relations.
+This, in turn, leads to "obvious" properties failing to hold. For instance, the expectation that observing a node's selection over time yields a strictly improving sequence may not hold, as different observers could disagree on whether each selection is "strictly better" than the previous one. This non-objectivity can have practical effects, particularly for diffusion pipelining, which relies on a clear, consistent chain order.
+- **Potential for Cycles**: The non-transitivity can conceptually give rise to "cycles" in preference, such as `A < B < C = A`. However, in practice, the node's implementation guarantees that it will never end up changing its selection in such a cycle because blocks already in the `VolatileDB` are not added again.
```

Comment:
It might be useful to mention here that the "Consistency with `Ord`" requirement of the `ChainOrder` class guarantees that we never can have `preferCandidate A B`, `preferCandidate B C` *and* `preferCandidate C A` (so no "cycles of preference").